Sources/Sentry/Public/SentryOptions.h

```diff
@@ -164,13 +159,6 @@ NS_SWIFT_NAME(Options)
  */
 @property (nonatomic, assign) BOOL stitchAsyncCode;
 
-/**
- * Describes the Sentry SDK and its configuration used to capture and transmit an event.
- * This is reserved for internal use, and will be removed in a future version of the SDK.
- */
-@property (nonatomic, readonly, strong) SentrySdkInfo *sdkInfo DEPRECATED_MSG_ATTRIBUTE(
-    "This property will be removed in a future version of the SDK");
```

Comment:
I chose to do it within the same PR as it was just removing some lines, so a small changes, that nevertheless would be annoying to do in its own PR because it was all touching the same code. It would have to be a branch of top of a branch, keep all that stuff in sync, deal with merge conflicts, etc.

Normally I definitely try to keep separate concerns in separate PRs!